source/local/local.go

```diff
@@ -41,6 +42,52 @@ type Source struct{}
 func (s Source) Name() string { return "local" }
 
 func (s Source) Discover() (source.Features, error) {
+	hooksFeatures, err := getFeaturesFromHooks()
+	if err != nil {
+		log.Printf(err.Error())
+	}
+
+	readsFeatures, err := getFeaturesFromReads()
+	if err != nil {
+		log.Printf(err.Error())
+	}
+
+	// Merge features from hooks and reads
+	for k, v := range hooksFeatures {
+		readsFeatures[k] = v
+	}
+
+	return readsFeatures, nil
+}
+
+// Format features to a valid format
+func formatFeatures(lines [][]byte, prefix string) source.Features {
```

Comment:
Would `parseFeatures` possibly be a more descriptive name?

Reply:
Good to me, I'll change this

Reply:
Done